src/test/java/api/loans/anonymization/AnonymizeLoansImmediatelyAPITests.java

```diff
@@ -0,0 +1,224 @@
+package api.loans.anonymization;
+
+import static api.support.matchers.LoanMatchers.isAnonymized;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.net.MalformedURLException;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import api.support.builders.LoanHistoryConfigurationBuilder;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeUtils;
+import org.junit.Test;
+
+public class AnonymizeLoansImmediatelyAPITests extends LoanAnonymizationTests {
+
+  /**
+   * Scenario 1
+   *
+   *     Given:
+   *         An Anonymize closed loans setting of "Immediately after loan closes"
+   *         An Anonymize closed loans with associated fees/fines setting of "Immediately after fee/fine closes"
+   *         An open loan with an associated fee/fine
+   *     When the item in the loan is checked in
+   *     Then do not anonymize the loan
+   */
+  @Test
+  public void testClosedLoanWithOpenFeesAndFinesIsNotAnonymizedForSettingsOfImmediately()
```

Comment:
Personally, I'd likely split this test into at least two tests, as I think there are two scenarios within this scenario:
* An open loan without any open fee / fine accounts does not get anonymized
* A closed loan with an open fee / fine account does not get anonymized